### PR TITLE
Added R examples from main

### PR DESCRIPTION
**Pull request by jnolis**
3 of the R examples (job, dashboards, and apis), were merged into main but not the release-2021.11.10 branch. This just brings them in.

### CODE REVIEW

File: examples/r-api/.saturn/saturn.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "example-r-api",
+  "image_uri": "saturncloud/saturn-rstudio:2021.11.10",
+  "description": "Deploy an API to Saturn Cloud using R .",
+  "working_directory": "/home/jovyan/examples/examples/r-api",
```
Comment (jsignell) [inline examples/r-api/.saturn/saturn.json]:
```suggestion
  "working_directory": "/home/jovyan/git-repos/examples/examples/r-api",
```

File: examples/r-job/.saturn/saturn.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "example-r-job",
+  "image_uri": "saturncloud/saturn-rstudio:2021.11.10",
+  "description": "Run a job on Saturn Cloud using R .",
+  "working_directory": "/home/jovyan/examples/examples/r-job",
```
Comment (jsignell) [inline examples/r-job/.saturn/saturn.json]:
```suggestion
  "working_directory": "/home/jovyan/git-repos/examples/examples/r-job",
```

File: examples/r-shiny/.saturn/saturn.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "example-r-shiny",
+  "image_uri": "saturncloud/saturn-rstudio:2021.11.10",
+  "description": "Deploy a dashboard to Saturn Cloud using R Shiny",
+  "working_directory": "/home/jovyan/examples/examples/r-shiny",
```
Comment (jsignell) [inline examples/r-shiny/.saturn/saturn.json]:
```suggestion
  "working_directory": "/home/jovyan/git-repos/examples/examples/r-shiny",
```

**Review by jsignell — CHANGES_REQUESTED** [review]
You need git-repos in the working dir on that release. Also did you mean to include any of these in the templates.json so that they show up in the resource creation bar?

**Comment by jnolis** [comment]
I put back the git-repos so those should be good. The template file is updated in a separate PR: https://github.com/saturncloud/examples/pull/245

**Review by jsignell — APPROVED** [review]
LGTM

**Pull request merged**